### DISCUSSION TITLE
Filename settings?  Video / Star trails

### DISCUSSION THREAD

**Discussion by tegwilym** [aaronwmorris/indi-allsky#1496] [General]
Hello!
I'm enjoying the software and should work well for my club observatory sky cam.  I'm running the StarlightXpress Oculus camera on a tripod in my yard for testing. 
I'm hoping to FTP the images up to the web for public viewing and have it running on a free web host page  -  http://dpas-skycam.free.nf/  nothing fancy just a free site that I can upload to.  Very ugly and just a test page currently.  Our club page is on WIX and they don't have FTP so I'm hoping to link them. 
So my question: 
I'm sure there is a way to do this on the web side (topic for another forum), but I can easily share a live view since it's named "image.jpg" which is easy to share on a web page.  
I'd like to share the daily time-lapse video when it's saved, but the file name is different every time and not sure how to save it as a common name like "video.mp4" or whatever.   I'm looking for video naming in the software, but figure it's embedded in a .conf file or something.  Is there a way to rename the saved file?  Even better if it can save one as video.mp4 and another with the full date/time stamp, but probably leads to web page work again.  I'd like to share maybe a week of videos that users can click and view, but that's been a challenge to figure out.   
Any tips on filenames?
Thanks,
Tom

**Comment by aaronwmorris** [comment]
I think I have another option for you instead of relying on static file names.  indi-allsky still creates files like "latest.jpg" but they are rarely ever used anymore.  With all of the image references stored in the database, the database may be queried for the latest image.  Even the Latest Image view does not use latest.jpg, it makes an AJAX call to the web instance which returns the latest image.

Since the image and video file names are unimportant, I still wanted to provide a way to reference these externally.  I came up with the idea of creating some redirect views which are static URLs which will return HTTP 302 redirects to the latest image or video.

https://github.com/aaronwmorris/indi-allsky/wiki/Redirect-Views

Merged #1500 and #1498 which has a redirect views for each asset type.

**Reply by aaronwmorris** [reply]
@tegwilym Are these new views able to work in your situation?

**Reply by tegwilym** [reply]
I still need to try it some more and read up on the merged postings.  I'm in a hold currently on the project. I'm ready to install it on the club observatory roof, then just work on the files remotely from home. Just need a path to run the USB through to the inside control room.  We have some ceiling leak work soon, so I plan on making a hole at that time.   
I have looked at Arduino ideas for turning the heat on and off depending on humidity, but that's another thing on my list!   :)
